utils/users.js

```diff
@@ -93,10 +93,34 @@ function getLowestLevelSkill(skills) {
   return { skill, level };
 }
 
+/**
+ * Creates pagination link for next and previous pages
+ *
+ * @param query {Object} - request query params
+ * @param cursor {string} - next | prev
+ * @param documentId {string} - firestore document Id
+ */
+
+function getPaginationLink(query, cursor, documentId) {
+  let baseUrl = "/users?";
+  if (Object.keys(query).length) {
```

Comment:
This can be done without:
-  `if-check`
```js
if(['next', 'prev', 'page'].includes(key)) continue;
```

Reply:
done